core/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id("org.jetbrains.kotlin.jvm")
+    id("com.google.protobuf")
```

Comment:
issue: Why do you need protobuf plugin in core module? I think we should remove it as this mode does not have ane gRPC